cmd/otelcol/config/collector/logs_config_linux.yaml

```diff
@@ -0,0 +1,740 @@
+  # See https://httpd.apache.org/docs/current/mod/mod_log_config.html#logformat
+  filelog/apache-access:
+    include: [ "/var/log/apache*/access.log", "/var/log/apache*/access_log", "/var/log/httpd/access.log", "/var/log/httpd/access_log" ]
+    operators:
+      - type: regex_parser
+        regex: '^(?P<host>.+) (?P<remote_logname>.+) (?P<user>.+) \[(?P<time>.+)\] "(?P<method>.+) (?P<path>.+) (?P<protocol>.+)" (?P<code>\d+) (?P<size>\d+) "(?P<referer>.+)" "(?P<agent>.+)"$'
+        timestamp:
+          parse_from: attributes.time
+          layout: '%d/%b/%Y:%H:%M:%S %z'
+    include_file_name: false
+    include_file_path: true
+    storage: file_storage/filelogs
+
+  # Apache error log.
+  # Configured using the ErrorLogFormat directive.
+  # By default, date and time are in the locale-specific representation (%c in strftime).
+  # See https://httpd.apache.org/docs/current/mod/core.html#errorlogformat
+  filelog/apache-error:
+    include: [ "/var/log/apache*/error.log", "/var/log/apache*/error_log", "/var/log/httpd/error.log", "/var/log/httpd/error_log" ]
+    operators:
+      - type: regex_parser
+        regex: '^\[(?P<time>.+?)\] \[(?P<module>\w+):(?P<level>\w+)\] \[pid (?P<pid>\d+):tid (?P<tid>\d+)\] (?P<log>.*)$'
+        timestamp:
+          parse_from: attributes.time
+          layout: '%c' # locale specific
+        severity:
+          parse_from: attributes.level
+          mapping:
+            fatal: emerg
+            error3: alert
+            error2: crit
+            info2: notice
+      - type: move
+        from: attributes.log
+        to: body
+      - type: retain
+        fields:
+          - attributes.module
+          - attributes["log.file.path"]
+    include_file_name: false
+    include_file_path: true
+    storage: file_storage/filelogs
+
+  # Apache Cassandra default log.
+  # Configured in /etc/cassandra/logback.xml, SYSTEMLOG appender section.
+  # Default date and time format is ISO-8601 (year-month-day hour:minute:second).
+  # See https://cassandra.apache.org/doc/latest/cassandra/troubleshooting/reading_logs.html
+  filelog/cassandra:
+    include: [ "/var/log/cassandra/cassandra.log", "/var/log/cassandra/system.log" ]
+    multiline:
+      line_start_pattern: '^[A-Z]+\s+\[[\w:]+\]\s\d'
+    operators:
+      - type: regex_parser
+        regex: '^(?P<level>\w+) +\[(?P<thread>[\w:]+)\] (?P<time>.+) (?P<source_file>\S+):(?P<source_line>\d+) - (?P<log>[\s\S]*)$'
+        timestamp:
+          parse_from: attributes.time
+          layout: "%Y-%m-%d %H:%M:%S"
+        severity:
+          parse_from: attributes.level
+      - type: move
+        from: attributes.log
+        to: body
+      - type: retain
+        fields:
+          - attributes.thread
+          - attributes["log.file.path"]
+    include_file_name: false
+    include_file_path: true
+    storage: file_storage/filelogs
+
+  # Apache Cassandra debug log. Contains additional debugging information.
```

Comment:
Since we have a very comprehensive config for different Cassandra logs here, do we want to include Garbage collection logs? It is usually written to `gc.log` file. It could be used as reference for other jvm services too.